### ISSUE TITLE
2.10.1 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3411]
- #3295
- #3297 
- #3299 
- #3363 
- #3393
- #3378 
- #3382
- #3395 
- #3373 
- #3397
- #3372 
- #3275 
- #3409 
- #3410

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick d3ef421524d4a601ec5d8eb357e4b68d1ebf8a76
git cherry-pick b379631f9093024cbfac4c4ee0ead392c309ea1f
git cherry-pick 041cc6cee6666db23c0850343d0c601f229c1b0b
git cherry-pick 1b175ea19531325f7585d585556d2ce751867cf7
git cherry-pick fc939bef4468c382c8f04d0a8a0003c41b4ad441
git cherry-pick 5846297f672139f4f7e329c7d6c940c1efb7122c
git cherry-pick a6ca1473ee7cf18b017475a77329f17f75c48fcc
git cherry-pick 86a9e18e90566932fa5486757ac56e2425ca4a99
git cherry-pick aaea1891d3a1519d71c9430f8183e2a1afac6981
git cherry-pick c4c084ef8804ff557e501e8f104f96dea5381a13
git cherry-pick c11275d7de2a525a060fed319dd4773391395724
git cherry-pick ed4a4b5f280714ab2ba90753b85927c98881d692
git cherry-pick e89db68dab645547998608c2aa1a9856a527b8be
git cherry-pick 44eb81511cedbf58279dc3dc9e8262604e39e8cb
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.10.0..HEAD --pretty='format:- %s'`
```
- Closes #3125: Update drupal/linkit requirement from 6.1.3 to 6.1.4 (#3410)
- Closes #3405: Add config_readonly patch for ConfigEntityListBuilder forms. (#3409)
- Closes #2735 Additional content type displays for AZ Side-by-Side Views (#3275)
- Update citation-style-language/styles requirement from 0.0.409 to 0.0.411 (#3372)
- Update Arizona Bootstrap to 2.0.27 (#3397)
- Update citation-style-language/locales requirement from 0.0.19 to 0.0.20 (#3373)
- Update drupal/core-recommended requirement from 10.2.5 to 10.2.6 (#3395)
- Update Arizona Bootstrap to 2.0.26 (#3382)
- Close #3377 Require "Access Draggableviews" permission to access carousel items draggable view. (#3378)
- Closes #3339 New breakpoint to hide arrows on small displays (#3393)
- Closes #3307: Add reset order button to draggable views. (#3363)
- Closes #3216 Adjust carousel fields vertical spacing (#3299)
- Closes #3214 Update carousel image help text (#3297)
- Closes #3218 Reduce carousel item width to 1110px (#3295)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.10.1
